### PR TITLE
New Sqlite Cheat Sheet

### PR DESCRIPTION
**Pull request by mohan08p**
IA page : https://duck.co/ia/view/sqlite3_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/sqlite.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
+{
+    "id": "sqlite3_cheat_sheet",
+    "name": "Sqlite3",
+    "description": "It is a C library that provides a lightweight disk-based database that doesn’t require a separate server process and access via sql",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/sqlite.json]: You shouldn't start the descriptions with `It is ...` -- The first sentence from Wikipedia does a good job of describing Sqlite: "Relational database management system contained in a C programming library"

File: share/goodie/cheat_sheets/json/sqlite.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    "metadata": {
+        "sourceName": "Cheatography",
+        "sourceUrl": "http://www.cheatography.com/richardjh/cheat-sheets/sqlite3/"
+    },
+    "aliases": [
+        "sqlite3 language", "sqlite3 functions" 
+    ],
+    "template_type": "code",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/sqlite.json]: This should be `terminal`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=sqlite3 cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/sqlite3_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@mohan08p This LGTM :+1:

Thanks again for taking the time to contribute! We really appreciate it :smiley:

**Comment by daxtheduck** [comment]
@mohan08p this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**